### DISCUSSION TITLE
3D Models pipeline

### DISCUSSION THREAD

**Discussion by adrianposteuca** [google/model-viewer#3583] [Q&A]
Hi all,
 I was wondering about your 3d Models pipeline for ModelViwer.
I haven't found a good pipeline so far. I am working in C4D so my models are TOO BIG even though I am working in CM. 
So I scale them down in Sketchfab and bring them to ModelViwer Editor.
I do not know what is wrong with C4D ( Any advice?)

Or another recommended pipeline?

Even now when I add them to my project some of the 3D models are huge.... how can I find a proper scale?
Any Advice?

**Comment by elalish** [comment]
Huge can mean a lot of things; do you mean too many MB, or the physical scale in AR? The glTF format is always in meters; other formats don't typically specify. You'll need to scale until your working units are meters so that you glTF export works properly (cm and mm are more popular in animation tools).
